public/index.html

```diff
@@ -2,7 +2,12 @@
 <head>
 </head>
 <body>
-  <section id="app">
-  </section>
+  <section id="app"></section>
+  <script src="https://unpkg.com/react@16/umd/react.development.js" crossorigin></script>
```

Comment:
I have never seen this before. Is this in place of adding the bundle script?
You may be onto something I gotta learn 😅